manifests/addons/broker-tls.yml

```diff
@@ -53,7 +53,3 @@ instance_groups:
     properties:
       broker:
         host: (( concat params.broker_domain ":443" ))
-
-meta:
-  broker:
-    protocol: https
```

Comment:
Why was this removed? This is used to tell the route_registrar what scheme to register with the gorouter